components/layout/Container/index.js

```diff
@@ -1,48 +1,34 @@
 import PropTypes from "prop-types";
-import styled from "styled-components";
 import classNames from "classnames";
-import {
-  containerNarrow,
-  containerRegular,
-  PADDING_LARGE,
-  tokens,
-} from "@/styles/globalStyles";
+import * as Styled from "./styles";
 
 export default function Container({
   bgColor = "white",
   children,
   className,
   width = "narrow",
+  paddingSize = "large",
   elAttributes,
 }) {
   return (
-    <Section className={className} bgColor={bgColor} {...elAttributes}>
-      <Inner
+    <Styled.Section
+      className={className}
+      $bgColor={bgColor}
+      $paddingSize={paddingSize}
```

Comment:
@dananjohnson Which do you prefer - using variables like this in styled components, or using the existing utility class `l-pad-top-${paddingSize}`? I prefer the utility class, but this component was already using variables.

Reply:
Utility class is good by me 👍🏻